Documentation/HelpInside/KeyboardCommands/Index.rst

```diff
@@ -0,0 +1,64 @@
+.. include:: /Includes.rst.txt
+
+.. _keyboard_commands:
+
+=================
+Keyboard Commands
+=================
+
+Navigating the Page Tree using a keyboard
+=========================================
+
+It is possible to navigate the page tree using only a keyboard and it adheres to
+the guidelines set out in `WAI-ARIA Authoring Practices 1.1. <https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction-22>`__
+
+* `Home` moves the focus to the first item in the page tree.
```

Comment:
For keys, I would recommend the use of the :kbd: textrole, see https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/InlineCode.html#using-text-roles

e.g. 

```
:kbd:`Home`
```

Reply:
Resolved - thank you.